src/main/resources/templates/home.html

```diff
@@ -3,9 +3,37 @@
 <head>
     <title>Aggregator</title>
     <meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />
+    <link rel="stylesheet" type="text/css" href="css/main.css">
```

Comment:
Strzelam, że sprawdzasz wygląd stron "na sucho", czyli po prostu odpalasz pliki html w przeglądarce i patrzysz czy dobrze to wygląda. Rozwiązanie jest jak najbardziej spoko, jeżeli chcesz sprawdzić kosmetyczne zmiany. 

Ale dla sprawdzenia jak serwer radzi sobie z twoimi widokami, musisz już normalnie odpalić go i wklepać np localhost:15001/ i wtedy zobaczysz stronę home.html zinterpretowaną przez serwer. 

I tu rodzą się pewne różnice.
1. Pliki css muszą być w odpowiednich folderach. Zamiast `resources/css/main.css`, musi on znaleźć się w `resources/static/css/main.css`
2. Obrazki też muszą znajdować się w `resources/static/img/...`

 **oczywiście te zmiany spowodują, że jak odpalisz stronę statycznie (czyli jako plik html) to nie znajdzie Ci ona css i obrazków. Więc tak w rzeczywistości możesz przenieść obrazki i cssy do poprawnych folderów dopiero pod koniec jak już wszystko będzie Ci odpowiadać. Wtedy tylko odpal serwer i zobacz czy wszystko poprawnie się wyświetla**